metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi-lexical-grammar.rule

```diff
@@ -426,7 +432,7 @@ each_expression:    "each" WS_INLINE each_expression_body
 
 each_expression_body:   function_body
 
-let_expression: "let" NEWLINE WS_INLINE? variable_list WS_INLINE? NEWLINE? in_expression
+let_expression: "let" whitespace variable_list WS_INLINE? NEWLINE? in_expression
```

Comment:
is this change safe?

Reply:
yes - this lets you put whitespace before/after the newline, and also lets you put `let Source = ...` on the same line

all of these are valid m-query so we have to match it